reference/test/factorization/cholesky_kernels.cpp

```diff
@@ -162,7 +162,7 @@ TYPED_TEST(Cholesky, KernelSymbolicCountSeparable)
          {0, 0, 0, 1, 1, 0, 0, 0, 0, 0},
          {0, 0, 0, 1, 1, 1, 0, 0, 0, 1},
          {0, 0, 0, 0, 1, 1, 0, 0, 0, 0},
-         {0, 0, 0, 0, 0, 0, 1, 0, 0, 1},
+         {0, 0, 0, 0, 0, 0, 1, 1, 0, 1},
```

Comment:
symmetric for cholesky? It's still correct in previous version because we only use the lower part?

Reply:
yes, precisely